server/model_test.go

```diff
@@ -133,3 +136,68 @@ The temperature in San Francisco, CA is 70°F and in Toronto, Canada is 20°C.`,
 		})
 	}
 }
+
+func TestParseFromFileFromLayer(t *testing.T) {
+	tempModels := t.TempDir()
+	t.Setenv("OLLAMA_MODELS", tempModels)
+	digest := "sha256-fb9d435dc2c4fe681ce63917c062c91022524e9ce57474c9b10ef5169495d902"
+
+	if _, err := GetBlobsPath(digest); err != nil {
```

Comment:
There's no reason to check this. I don't think this does what you think it does. `GetBlobsPath` only resolves a digest to a path but doesn't check if the blob itself exists.